gulpfile.js

```diff
@@ -25,6 +27,50 @@ gulp.task('test', ['build'], function () {
         });
 });
 
-gulp.task('default', function () {
-    return gulp.run('test');
+// All the dotnet tasks
+gulp.task('clean/dotnet', function () {
+    console.log('Cleaning build directories...');
+    return gulp.src(['src/dotnet/**/bin', 'src/dotnet/**/obj'], { read: false })
+        .pipe(clean({ force: true }));
+});
+
+gulp.task('restore/dotnet', ['clean/dotnet'], function () {
+    console.log('Running dotnet restore...');
+    return gulp.src('src/dotnet/**/*.csproj')
+        .pipe(restore());
+});
+
+gulp.task('build/dotnet', ['restore/dotnet'], function () {
+    console.log('Running dotnet build...');
+    return gulp.src('src/dotnet/**/*.csproj')
```

Comment:
Yes we do, but restore works on the csprojs and I thought we could keep this consistent